lib/msal-common/src/network/INetworkModule.ts

```diff
@@ -30,6 +30,7 @@ export interface INetworkModule {
      */
     sendGetRequestAsync<T>(
         url: string,
+        correlationId: string,
         options?: NetworkRequestOptions,
```

Comment:
2 ideas of how to make this change simpler: 

1. Add the correlationId as one of the options, as adding a prop to an object is non-breaking change
2. Add logging around HttpClient (granted this will provided limited visiblity into the http calls)

Reply:
The problem with 1 is that correlationID is required for the logger, so it can't be passed into an optional NetworkRequestOptions. With your above feedback in mind, it might be best to just abandon this PR. I don't think 2 will provide much visibility, like you said. The best course of action would then be to abandon this and keep telling customers to create their own INetworkModule.

Reply:
+1. I commented something similar above. Make it an optional param and the last one so we are not messing up the existing API.